lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -204,7 +205,8 @@ tokenize([$~, S, H | T] = Original, Line, Column, Scope, Tokens) when ?is_sigil(
   case elixir_interpolation:extract(Line, Column + 3, Scope, ?is_downcase(S), T, sigil_terminator(H)) of
     {NewLine, NewColumn, Parts, Rest} ->
       {Final, Modifiers} = collect_modifiers(Rest, []),
-      Token = {sigil, {Line, Column, nil}, S, tokens_to_binary(Parts), Modifiers, <<H>>},
+      Indentation = nil,
```

Comment:
I wasn't sure what to put into non-heredoc sigils, either this (so we don't put indentation into meta) or maybe `Indentation = Column + 3`, the start of the sigil string.

Reply:
`nil` sounds good to me.